pkg/ui/react-app/src/pages/alerts/AlertContents.tsx

```diff
@@ -54,49 +77,80 @@ const AlertsContent: FC<AlertsProps> = ({ groups = [], statsCount }) => {
     });
   };
 
+  const handleSearchChange = (e: ChangeEvent<HTMLTextAreaElement | HTMLInputElement>) => {
```

Comment:
`persist()` missing (:

Reply:
I put it in the global component `SearchBar` (see [46031b7](https://github.com/thanos-io/thanos/pull/5119/commits/46031b76c0e85e761861530eb68079a7a0fcc4d0)). 

Hopefully that's the only place it's required. The function you are looking here is called by the search component where you have the debounce mechanism. So it's in this component that you should persist the event 

On my side it's working, is it still broken on your side @GiedriusS